boot/zephyr/include/sysflash/sysflash.h

```diff
@@ -3,10 +3,21 @@
 #ifndef __SYSFLASH_H__
 #define __SYSFLASH_H__
 
+#ifdef USE_PARTITION_MANAGER
+#include <pm_config.h>
+
+#define FLASH_AREA_IMAGE_PRIMARY    PM_CFG_MCUBOOT_PARTITIONS_PRIMARY_ID
```

Comment:
I would omit the `_CFG_` infix; don't find it very meaningful.

Reply:
will remove